sdk/storage/azure-storage-blob/azure/storage/blob/_upload_helpers.py

```diff
@@ -318,7 +330,7 @@ def upload_append_blob(  # pylint: disable=unused-argument
                 headers=headers,
                 **kwargs)
         except HttpResponseError as error:
-            if error.response.status_code != 404:
+            if error.response.status_code != 404:  #type: ignore
```

Comment:
azure\storage\blob\_upload_helpers.py:333: error: Item "None" of "Optional[_HttpResponseCommonAPI]" has no attribute "status_code"  [union-attr]